README.md

```diff
@@ -244,6 +244,24 @@ You can access nested keys in records via dot or bracket notation (https://docs.
 
 See Supported Metric Type and Labels for more configuration parameters.
 
+### prometheus_pushgateway output plugin
+
+You can configure this plugin to push metrics to [Pushgateway](https://github.com/prometheus/pushgateway) collected by other Prometheus plugins.
+With following configuration, you can push data.
+
+```
+<match>
+  @type prometheus_pushgateway
+</match>
```

Comment:
`job_name` is required parameter so it should be written here.

Reply:
fixed! https://github.com/fluent/fluent-plugin-prometheus/pull/134/files#diff-04c6e90faac2675aa89e2176d2eec7d8R255